stan/math/rev/scal/fun/frexp.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef STAN_MATH_REV_SCAL_FUN_FREXP_HPP
+#define STAN_MATH_REV_SCAL_FUN_FREXP_HPP
+
+#include <stan/math/rev/core.hpp>
+#include <cmath>
+#include <limits>
+
+namespace stan {
+  namespace math {
+
+	/*
+	 * The derivative of frexp is 0, because frexp returns
```

Comment:
This doc should start with a sentence about what the class is for.  Here, it can say something simple, like `Variable implementation for frexp.`  This is what will show up in the one-line summary in the generated doc.